components/elm/src/main/elm_driver.F90

```diff
@@ -231,6 +256,14 @@ subroutine elm_drv(doalb, nextsw_cday, declinp1, declin, rstwr, nlend, rdate)
     character(len=256)   :: dateTimeString
     type(bounds_type)    :: bounds_clump
     type(bounds_type)    :: bounds_proc
+#if defined(CLDERA_PROFILING)
+    integer :: ncols, part_dim, nparts, part_alloc_size
+    integer :: dims(3)
+    integer :: begp, endp
+    character(len=4) :: dimnames(3)
+    real(r8), pointer :: field1d(:)
```

Comment:
some of this is probably not needed either?

Reply:
yep, only needed the ymd. Thanks!